pkg/image/api/helper.go

```diff
@@ -944,6 +948,20 @@ func PrioritizeTags(tags []string) {
 				minor = append(minor, v)
 				continue
 			}
+		case reMajorWithPatch.MatchString(short):
```

Comment:
semver is only capable of working with fully specified version string iow. `x.y.z` with or without `-patch` part.